benches/store_bench.rs

```diff
@@ -13,74 +17,94 @@ fn create_temp_directory() -> TempDir {
 }
 
 fn bulk_insert(c: &mut Criterion) {
-    let mut count = 0_u32;
-    let mut bytes = |len| -> Vec<u8> {
-        count += 1;
+    let count = AtomicU32::new(0_u32);
+    let bytes = |len| -> Vec<u8> {
         count
+            .fetch_add(1, Relaxed)
             .to_be_bytes()
             .into_iter()
             .cycle()
             .take(len)
-            .clone()
             .collect()
     };
 
     let mut bench = |key_len, val_len| {
+        let rt = tokio::runtime::Builder::new_multi_thread()
```

Comment:
Probably doesnt need to be addressed in this PR, but was wondering if this doesnt usually get injected via the harness. Given its a benchmark then probably better to keep full control over it. But sharing the idea for consideration.

Reply:
Thanks for the info @phughk, I wasn't aware, I'll look into how this is done

Reply:
`Tokio` will park it's threads if it's not being used. Therefore, you can have two runtimes. If one runtime is no longer being utilised, it will park all it's threads.